src/components/Card/Card.Styled.ts

```diff
@@ -35,23 +35,23 @@ export const Text = styled.div<{ isActive?: boolean }>`
 	right: 0;
 	bottom: 0;
 	padding: 6.5em 2em;
-	background-color: ${(props) => props.theme.colors.almostDark};
-	color: ${(props) => props.theme.colors.pureWhite};
+	background-color: ${({ theme }) => theme.colors.almostDark};
+	color: ${({ theme }) => theme.colors.pureWhite};
 	clip-path: ${({ isActive }) =>
 		isActive ? "circle(120% at 50% 90%)" : "circle(0 at 50% 90%);"};
```

Comment:
Tutaj w sumie całość jest taka sama, jedynie różni się pierwszy parametr funkcji circle, może to ją wyifować zamiast całości?